src/integration/helpers/testHelper.ts

```diff
@@ -1,12 +1,15 @@
-/* 
-import { screen } from "@testing-library/react";
+
+import { screen, within } from "@testing-library/react";
 import userEvent from "@testing-library/user-event";
- */
+
 /* INFO AREA */
 export const INFO_AREA_ID = "INFO_AREA";
-export const NEXT_MONTH_ID = "NEXT_MONTH";
-export const PREV_MONTH_ID = "PREV_MONTH";
+export const BUTTON_NEXT_MONTH_ID = "NEXT_MONTH";
```

Comment:
From now on, since we are using TS now, you can specify the type of the ids, that are strings in this way:

```suggestion
export const BUTTON_NEXT_MONTH_ID: string = "NEXT_MONTH";
```